test/e2e/system_connection_test.go

```diff
@@ -71,6 +71,8 @@ var _ = Describe("podman system connection", func() {
 			session.WaitWithDefaultTimeout()
 			Expect(session).Should(Exit(0))
 			Expect(session.Out.Contents()).Should(BeEmpty())
+			// stderr will probably warn (ENOENT or EACCESS) about socket
+			// but it's too unreliable to test for.
```

Comment:
I really dislike this. We can't use `ExitCleanly()` here because, although the command exits 0, it spits out ugly warnings on both root and rootless.